crypto/http/http_local.h

```diff
@@ -16,18 +16,18 @@
 /* name aliases for legacy names with name prefix "OCSP_" */
 typedef OCSP_REQ_CTX OSSL_HTTP_REQ_CTX;
 /* functions meanwhile only used internally */
-# define OSSL_HTTP_REQ_CTX_new          OCSP_REQ_CTX_new
-# define OSSL_HTTP_REQ_CTX_free         OCSP_REQ_CTX_free
-# define OSSL_HTTP_REQ_CTX_header       OCSP_REQ_CTX_http
-# define OSSL_HTTP_REQ_CTX_add1_header  OCSP_REQ_CTX_add1_header
-# define OSSL_HTTP_REQ_CTX_i2d          OCSP_REQ_CTX_i2d
-# define OSSL_HTTP_REQ_CTX_nbio         OCSP_REQ_CTX_nbio
-# ifndef OPENSSL_NO_SOCK
-#  define OSSL_HTTP_REQ_CTX_sendreq_d2i OCSP_REQ_CTX_nbio_d2i
-# endif
+#  define OSSL_HTTP_REQ_CTX_new         OCSP_REQ_CTX_new
+#  define OSSL_HTTP_REQ_CTX_free        OCSP_REQ_CTX_free
+#  define OSSL_HTTP_REQ_CTX_header      OCSP_REQ_CTX_http
+#  define OSSL_HTTP_REQ_CTX_add1_header OCSP_REQ_CTX_add1_header
+#  define OSSL_HTTP_REQ_CTX_i2d         OCSP_REQ_CTX_i2d
+#  define OSSL_HTTP_REQ_CTX_nbio        OCSP_REQ_CTX_nbio
+#  ifndef OPENSSL_NO_SOCK
+#   define OSSL_HTTP_REQ_CTX_sendreq_d2i OCSP_REQ_CTX_nbio_d2i
+#  endif
```

Comment:
These extra indents don't seem to be needed??

Reply:
Right, this was a mistake: At first I thought that these #defines had to be put under `#ifndef OPENSSL_NO_OCSP` but when I removed that again I forgot to un-indent them. Fixed.